zephyr/user_settings.h

```diff
@@ -0,0 +1,153 @@
+/* user_settings.h
```

Comment:
Please use https://github.com/wolfSSL/wolfssl/blob/master/examples/configs/user_settings_wolftpm.h as template for this user_settings.h. Thank you